lib/puma/dsl.rb

```diff
@@ -251,6 +251,10 @@ def bind_to_activated_sockets(bind=true)
       @options[:bind_to_activated_sockets] = bind
     end
 
+    def events(events)
+      @options[:events] = events
+    end
```

Comment:
Do we need to add `events` to the DSL? Should we document it? Looks like some kind of hack to me? :) Should it be a public thing in the DSL?

Reply:
Should the other DSL objects also have access to `@events`?

https://github.com/puma/puma/blob/5f255fcd132a3d67608a3d82b31c540bdbf69bcf/lib/puma/configuration.rb#L146-L148

Reply:
It was long time ago, but I think it was result of this discussion: https://github.com/puma/puma/pull/2709#discussion_r713992970

It was made as a temporary measure, to be refactored later I guess. I'm not sure if that is still right approach

Reply:
I have changed the behavior, so now there is no need for this method and the `Puma::Events` instance is passed early to `Puma::Configuration` so it can be used by any type of DSL if needed